.github/CODEOWNERS

```diff
@@ -1,9 +1,9 @@
 # Global Reviewers
 # review when someone opens a pull request.
-*       @cartyc @fmichaelobrien
+*       @tackaberry @fmichaelobrien 
 
 # CLI Reviewers
 cli/*   @shaunmitchellve
 
 # Solution Reviewers
-solutions/* @davelanglois-ssc @alaincormier-ssc
\ No newline at end of file
+solutions/* @davelanglois-ssc @alaincormier-ssc
```

Comment:
there's an old/outdated PR #281 to add some solutions reviewers
for now, we could add @lucstjean-ssc and @stephanemillaire-ssc

Reply:
adding and closing #281